content/awell-orchestration/api-reference/mutations/schedule-track.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Schedule track
+description: Schedule a track in a patient care flow (pathway)
```

Comment:
As long as we don't have a V2 of our API that adopts "care flow", I prefer we use pathway consistently in the documentation.

```suggestion
description: Schedule a track in a patient pathway
```